source/manual/2nd-line-drills.html.md

```diff
@@ -81,9 +81,47 @@ See "[If Slack is unavailable](https://docs.google.com/document/d/144y8c2Ly-kG3J
 In preparation for a spike in traffic, you can increase the number of unicorn workers for an app.
 See [established connections exceeded](/manual/alerts/established-connections-exceed.html) for details.
 
-Pick an application and drill scaling up the number of workers - see [example](https://github.com/alphagov/govuk-puppet/pull/11194).
+Pick an application and drill scaling up the number of workers.
 
-You can create a branch of `govuk-puppet` and deploy that branch to Integration to see the unicorn worker change take effect. Delete the branch and re-deploy the latest release of Puppet when you're done.
+### Check the number of unicorn workers currently running
+
+You will need to SSH into the machine class for your chosen application, you can find this information in the developer docs, for example, the machine class for [Smart Answers](https://docs.publishing.service.gov.uk/repos/smart-answers.html) is `calculators_frontend`
+
+SSH into the machine on Integration using the command below
+
+`gds govuk connect -e integration ssh calculators_frontend`
+
+Once connected, run the command below to get a list of the Unicorn Workers running on the machine:
+
+`ps -ax | grep unicorn | grep smartanswers`
+
+You should then get an output similar to that below. In this example we can see that 4 unicorn workers are running on the machine.
+
+```shell
+18789 ?        S      0:00 unicorn master -D -P /var/run/smartanswers/app.pid -p 3010 -c /var/apps/smartanswers/config/unicorn.rb
+18790 ?        S      0:12 unicorn worker[0] -D -P /var/run/smartanswers/app.pid -p 3010 -c /var/apps/smartanswers/config/unicorn.rb
+18791 ?        S      0:11 unicorn worker[1] -D -P /var/run/smartanswers/app.pid -p 3010 -c /var/apps/smartanswers/config/unicorn.rb
+18792 ?        S      0:12 unicorn worker[2] -D -P /var/run/smartanswers/app.pid -p 3010 -c /var/apps/smartanswers/config/unicorn.rb
+18793 ?        S      0:13 unicorn worker[3] -D -P /var/run/smartanswers/app.pid -p 3010 -c /var/apps/smartanswers/config/unicorn.rb
+```
+
+### Increase the number of unicorn workers
+
+- Create a branch of [govuk-puppet](https://github.com/alphagov/govuk-puppet)
+- Increase the number of unicorn workers for your chosen application
+  - [Example PR](https://github.com/alphagov/govuk-puppet/pull/11194) - Please note, the example shown is for production. For the drill, the change will need to be made in `integration.yaml`
+- [Deploy your branch to Integration](https://deploy.integration.publishing.service.gov.uk/job/Deploy_Puppet/)
+- Run `govuk_puppet --test` to pull in the changes
+- Run `ps -ax | grep unicorn | grep smartanswers`
+- Check the number of unicorn workers listed is as expected
+
+### Revert the changes made on Integration
+
+- Re-deploy the latest release of Puppet
+- Run `govuk_puppet --test` to pull in the changes
+- Run `ps -ax | grep unicorn | grep smartanswers`
+- Check the number of unicorn workers listed are as before
+- Delete your test branch
```

Comment:
I think you can delete this subsection (I've tried to cover this in the first comment of the review, feel free to add more detail there if you feel it's necessary).

Reply:
I've removed this subsection.